src/platform/forms-system/src/js/widgets/TextWidget.jsx

```diff
@@ -23,6 +23,7 @@ export default function TextWidget(props) {
     onBlur: () => props.onBlur(props.id),
     onChange: event =>
       props.onChange(event.target.value ? event.target.value : undefined),
+    onFocus: props.onFocus,
```

Comment:
Bonus points for adding it to `propTypes`, but I see that like none of these props are there, so I guess kinda whatever.